core/module.h

```diff
@@ -72,16 +60,15 @@ struct PbCommand {
   int mt_safe;
```

Comment:
In the new bessctl.cc, there is no logic to check if a command is multi-thread safe.